lib/nadia/api.ex

```diff
@@ -54,7 +55,7 @@ defmodule Nadia.API do
   * `file_field` - specify the key of file_field in `options` when sending files
   """
   def request(method, options \\ [], file_field \\ nil) do
-    timeout = ((options[:timeout] || @default_timeout) + 1) * 1000
+    timeout = ((options[:timeout] || 0) + (opt_recv_timeout || @default_timeout)) * 1000
```

Comment:
a little bit complicated…
I'd like to get the fallback value in `opt_recv_timeout` method.

And I also prefer `get(source, key, deafult_value)` to 
`|| default_value`.
So I'll write like this:
```elixir
defp recv_timeout, do: Application.get_env(:nadia, :recv_timeout, @default_timeout)

timeout = (Keyword.get(options, :timeout, 0) + recv_timeout) * 1000
```